### PR TITLE
[BULK UPDATE] DocuTune - Fix build validation issues: docs-link-absolute (part 2)

### PR DESCRIPTION
**Pull request by alexbuckgit**
[BULK UPDATE] DocuTune - Fix build validation issues: docs-link-absolute

**Global effort to fix build validation errors**

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR was generated using DocuTune. It includes only build validation fixes and does not change other content.

DocuTune PRs improve quality of technical content by finding and automatically correcting a broad set of issues only where the correction is suitable. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

InitiativeId: docutune-build-validation-issues-2022-05
PointOfContact: @MonicaRush
CorrelationId: 0df9c2d6-63f8-4d1e-8b57-b33611a06514

### CODE REVIEW

File: docs/outlook/outlook-home.md [hunk 1]
```diff
@@ -13,32 +13,31 @@ ms.localizationpriority: high
 Find how-to content, sample code, SDK and API documentation, VBA references, training, and technical articles for developing solutions and customizing Outlook. 
 
 > [!NOTE]
-> Interested in developing solutions that extend the Office experience across [multiple platforms](https://docs.microsoft.com/office/dev/add-ins/overview/office-add-in-availability)? Check out the new [Office Add-ins model](https://docs.microsoft.com/office/dev/add-ins/overview/office-add-ins). 
+> Interested in developing solutions that extend the Office experience across [multiple platforms](/javascript/api/requirement-sets)? Check out the new [Office Add-ins model](https://docs.microsoft.com/office/dev/add-ins/overview/office-add-ins).
```
Comment (MeeraDi) [inline docs/outlook/outlook-home.md:16]: The Office Add-ins model link was skipped.

File: docs/outlook/outlook-home.md [hunk 2]
```diff
@@ -13,32 +13,31 @@ ms.localizationpriority: high
...
-- [Office VBA Object library reference](https://docs.microsoft.com/office/vba/api/overview/library-reference)
+- [Office VBA Object library reference](/office/vba/api/overview/library-reference)
   
-- [Office VBA language reference](https://docs.microsoft.com/office/vba/api/overview/language-reference)
+- [Office VBA language reference](/office/vba/api/overview/language-reference)
   
-- [Open XML SDK](https://docs.microsoft.com/office/open-xml/open-xml-sdk)
+- [Open XML SDK](/office/open-xml/open-xml-sdk)
   
 - [VSTO Add-ins for Office](https://docs.microsoft.com/visualstudio/vsto/create-vsto-add-ins-for-office-by-using-visual-studio?view=vs-2017)
```
Comment (MeeraDi) [inline docs/outlook/outlook-home.md:42]: The VSTO Add-ins for Office link was missed.

File: docs/outlook/pia/how-to-get-the-global-address-list-or-a-set-of-address-lists-for-a-store.md [hunk 3]
```diff
@@ -17,11 +17,11 @@ This topic contains two code examples that show how to get the Global Address Li
...
+GetGlobalAddressList uses the [PropertyAccessor](https://docs.microsoft.com/dotnet/api/microsoft.office.interop.outlook.propertyaccessor?redirectedfrom=MSDN&view=outlook-pia) object and the MAPI property https://schemas.microsoft.com/mapi/proptag/0x3D150102 to obtain the PR\_EMSMDB\_SECTION\_UID property of an address list and the current store. GetGlobalAddressList identifies an address list as associated with a store if their PR\_EMSMDB\_SECTION\_UID properties match, and the address list is the Global Address List if its [AddressListType](#Microsoft_Office_Interop_Outlook_AddressList_AddressListType) property is [olExchangeGlobalAddressList](/dotnet/api/microsoft.office.interop.outlook.oladdresslisttype?redirectedfrom=MSDN&view=outlook-pia). If the call to GetGlobalAddressList succeeds, DisplayGlobalAddressListForStore uses the [SelectNamesDialog](/dotnet/api/microsoft.office.interop.outlook.selectnamesdialog?view=outlook-pia&viewFallbackFrom=outlook-pia%2F&redirectedfrom=MSDN) object to display the returned Global Address List in the **Select Names** dialog box.
```
Comment (MeeraDi) [inline docs/outlook/pia/how-to-get-the-global-address-list-or-a-set-of-address-lists-for-a-store.md:24]: The Property/Accessor link was skipped

**Comment by MeeraDi** [comment]
#docutune-wait

File: docs/outlook/outlook-home.md (hunk 1 above)
Comment (alexbuckgit) [reply docs/outlook/outlook-home.md]: Corrected

File: docs/outlook/pia/how-to-get-the-global-address-list-or-a-set-of-address-lists-for-a-store.md (hunk 3 above)
Comment (alexbuckgit) [reply docs/outlook/pia/how-to-get-the-global-address-list-or-a-set-of-address-lists-for-a-store.md]: Corrected

**Comment by alexbuckgit** [comment]
#docutune-ready

**Comment by alexbuckgit** [comment]
@AlexJerabek @dstrome

Please merge the changes in this PR. For more information, see the description at the top of this PR.

Any pre-existing build validation issues or insufficient Acrolinx scores should be addressed in a separate PR.

#docutune-request-approval

**Comment by alexbuckgit** [comment]
Hi @lindalu-MSFT -
This PR is ready to merge. Thanks!

**Pull request merged**